app/services/team_service.rb

```diff
@@ -28,9 +27,9 @@ def deactivate(email, name)
     end
     if user
       response = slack_client.admin_set_inactive(user: user['id'])
-      Rails.logger.info "#{@team.space.subdomain}: deactivated user #{email}/#{name}: #{response}"
```

Comment:
again would be nice to have the member id here. slack ids don't help us much as we can't reference them anywhere.